Upgrade-Guide/op_version.md

```diff
@@ -10,7 +10,11 @@ After Gluster upgrade, it is advisable to have op-version updated.
 
 Current op-version can be queried as below:
 
-    [root@~]#gluster volume get  <volume name> cluster.op-version
+    [root@~]#gluster volume get all cluster.op-version
+
+To check maximum op-version supported by the cluster, the following query can be used:
```

Comment:
I felt this meant the same though. But I do see how it could be confusing. Will reword this to make it clearer.